### DISCUSSION TITLE
Security doors - off the shelf vs custom

### DISCUSSION THREAD

**Discussion by hernessteve** [NAIST-SE/AttractingContributors#1225] [General]
A friend of mine has about a 4,500 Sq Ft home that has a poor quality interior door going from the 3 car garage to the kitchen.

It is relatively easy to open a garage door these days.

Are there better quality ‘off the shelf’ security doors available or is custom the only solution?

The application would be to replace the interior door going from his garage interior to kitchen.

Thanks in advance.

**Comment by ellielo** [comment]
Assuming an inswing door into the house, first point of failure is generally the latch/lock route in the jamb blowing out. After that it can be the hinge screws failing or the panel at the knob/deadbolt mortise breaking.

Lots of metal jamb reinforcements on the market to beef up the existing jamb. Fixing the next two points of failure likely requires a new panel. Lots of heavy, solid core panels taken out of commercial buildings at salvage/restore places which can generally be prepped to match the existing hinge routes. A new steel panel and frame for this application isn't that expensive either if it's a standard size.

It's all for naught though if the door isn't consistently locked. And if the bad guys or gals got that far, they're now in a garage, probably able to close the doors and work to open this door in relative peace with the extra tools of what's there. A handyman jack placed horizontally at the latch to spread the jamb will quickly open most doors in wood framed structures

**Comment by MirkosMangle** [comment]
Look for doors that feature reinforced frames and locking mechanisms. Some models come with additional features such as multiple locks or heavy-duty hinges that enhance security. I worked with the [vosker customer service](https://www.pissedconsumer.com/company/vosker/customer-service.html) , which is a decent security company, you might wanna ask them for advice. Those guys know everything about safety
